sources/packages/backend/apps/db-migrations/src/sql/Messages/Create-messages.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS sims.messages (
+    id INTEGER PRIMARY KEY,
```

Comment:
Please use the same alias type used all around the file: `INT` instead of `INTEGER`.